docs/articles/documentation/guides/basic-guides/interact-with-the-page.md

```diff
@@ -193,12 +193,20 @@ test('Drag test', async t => {
 
 ## Upload Files
 
-Actions that allow you to interact with file upload input elements.
+TestCafe can't test native browser dialogs, including those during files selection. This section describes how to prevent these dialogs and select files for upload.
```

Comment:
1) Let's put this to a `### Prevent File Select Dialog` subsection after the example, can we?
2) How about phrasing these two sentences without starting  right away with `TestCafe cannot`: 

```
This section describes how to prevent file select dialogs and specify files for upload in code.

You must suppress file select dialogs because TestCafe cannot close them or otherwise interact with them.

...
```

Reply:
edited